### PR TITLE
add simple repl

### PR DESCRIPTION
**Pull request by deoxxa**

### CODE REVIEW

File: repl/repl.go [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+	"github.com/robertkrimen/otto"
+	"gopkg.in/readline.v1"
+)
+
+var i int
+
+// DebuggerHandler implements otto's debugger handler signature, providing a
+// simple drop-in debugger implementation.
+func DebuggerHandler(vm *otto.Otto) {
+	i++
```
Comment (darkliquid) [inline repl/repl.go]: Not sure how often people will be running this in a multi-threaded context, but this doesn't look thread-safe. Maybe use atomic.AddUint32 here.

Comment (deoxxa) [inline repl/repl.go]: Good catch - will fix.

**Comment by darkliquid** [comment]
Nice. Looks good. :+1:

**Comment by deoxxa** [comment]
Fun fact: the startup time for otto is way faster than node.

```
➜  otto git:(add-repl) ✗ time node example.js
Hello, person one!
Hello, person two!
node example.js  0.09s user 0.01s system 98% cpu 0.108 total
➜  otto git:(add-repl) ✗ time ./otto example.js
Hello, person one!
Hello, person two!
./otto example.js  0.00s user 0.01s system 72% cpu 0.012 total
```

**Comment by deoxxa** [comment]
Oh, I didn't even realise there was already an `otto` command. I better split this up.

**Comment by deoxxa** [comment]
...erm, wat.

**Pull request merged**